docs/6.x/config.md

```diff
@@ -10,7 +10,7 @@ Resource Name             | Resource Description
 `github`                  | GitHub connector
 `oidc`                    | OIDC connector (Enterprise only)
 `saml`                    | SAML connector (Enterprise only)
-`role`                    | Cluster role
+`role`                    | Cluster role (Enterprise only)
```

Comment:
Nit: "User role" or "Cluster user role" would be a bit more accurate? Also, since you're here anyway, could you please fix inconsistent capitalization of first words in the descriptions in this table? :)